opennms-container/minion/confd/templates/system.properties.tmpl

```diff
@@ -0,0 +1,161 @@
+################################################################################
```

Comment:
Can we update the default/existing system.properties files to include a statement like `${optionals} = confd.system.properties`  and append the properties there rather than templating this file?

That would save us from having to maintain this template for Karaf upgrades going forward.

Reply:
Yes, that should work. Do we have to worry about duplicates in that case? Or would we treat the values that are currently in the file as non-configurable by confd.

Reply:
Karaf documents this as "Included files will override the values specified in this file", so shouldn't have to worry about dupes.